core/services/ocr2/plugins/ocr2keeper/evm21/block_subscriber.go

```diff
@@ -18,107 +18,96 @@ import (
 )
 
 const (
-	// PollLogInterval is the interval to query log poller
-	PollLogInterval = time.Second
-	// CleanUpInterval is the interval for cleaning up block maps
-	CleanUpInterval = 15 * time.Minute
-	// ChannelSize represents the channel size for head broadcaster
-	ChannelSize = 20
+	// cleanUpInterval is the interval for cleaning up block maps
+	cleanUpInterval = 15 * time.Minute
+	// channelSize represents the channel size for head broadcaster
+	channelSize = 20
```

Comment:
what does this indicate? Is it the max blocks stored in the channel via head broadcaster while it waits to be read by this component?

Reply:
yes